pkg/util/validation/validate.go

```diff
@@ -68,6 +68,15 @@ func metricReasonFromErrorID(id globalerror.ID) string {
 	return strings.ReplaceAll(string(id), "-", "_")
 }
 
+// DiscardedRequests is a metric of the number of discarded requests.
+var DiscardedRequests = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "cortex_discarded_requests_total",
+		Help: "The total number of requests that were discarded due to rate limiting.",
+	},
+	[]string{"user"},
```

Comment:
Nit: should we include "reason" label for consistency with discarded samples and exemplars metrics?

Reply:
Good point! Agree. We should use the same reason used for samples rate limiting.